### DISCUSSION TITLE
Welcome to senselab Discussions!

### DISCUSSION THREAD

**Discussion by satra** [sensein/senselab#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adi611** [comment]
Hey everyone, I’m Aditya, currently working as a backend developer at Pocket FM. I’m looking to dive into research and broaden my learning, and thanks to @satra I’ve found this project. Looking forward to taking on challenges, learning new things, and solving problems. I’ve already started with task #34. Outside of work, I enjoy playing cricket and Age of Empires :)
